notification-portlet-webcomponents/notification-icon/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "notification-icon",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --name notificaiton-icon --target wc",
```

Comment:
I think this needs `--dest build/static/js` such that the resulting js gets picked up by Gradle

Reply:
I'll get this updated.
Heads up, in the not-too-distant future these components will be moved to their own repo and released as webjars.